tests/unit/test_modulegraph/test_modulegraph.py

```diff
@@ -465,6 +465,9 @@ def assertNoMethods(self, klass):
         if '__dict__' in d:
             # New in Python 3.4
             del d['__dict__']
+        if '__weakref__' in d:
+            # New in Python 3.5
+            del d['__weakref__']
```

Comment:
Please describe in the commit-message hwy you are doing this. (This become clear when reading the code (do make `assertEqua()` pass), but this should be somehow clear from the commit message.
Something like "… to make tests pass in Py > 3.4"